zinit-autoload.zsh

```diff
@@ -3246,7 +3246,8 @@ EOF
         setopt localoptions extendedglob nokshglob noksharrays
         builtin cd -q "${ZINIT[SNIPPETS_DIR]}"
         local -a list
-        list=( "${(f@)"$(LANG=en_US.utf-8 tree -L 3 --charset utf-8)"}" )
+        local -x LANG=en_US.utf-8
```

Comment:
The [prior check](https://github.com/zdharma-continuum/zinit/pull/190/files#diff-6a3752f1e40118bbfc5b0e05b9fa86aaffbd897b4c36f760bb3221a862f68343R3240-R3244) evaluates to false even though I have exa, but no tree.